src/EA.Weee.Web/Controllers/OrganisationRegistrationController.cs

```diff
@@ -624,6 +625,44 @@ private object CastToSpecificViewModel(ExternalOrganisationType? organisationTyp
             }
         }
 
+        [HttpPost]
+        [ValidateAntiForgeryToken]
+        public async Task<ActionResult> PartnerDetails(PartnerViewModel model, string action)
+        {
+            if (!ModelState.IsValid)
```

Comment:
@seanciv Noticed this AC
"Then the following fields are mandatory: 'First name', 'Last name' for the first 2 partners (for partner 3 onwards, the fields are optional)" is this covered? Might need an alternative PartnerModel model that doesnt have those fields as required. Could either add an additional property to PartnerViewModel to store the non mandatory field ones or define some interface between the two alternative PartnerModel types...  thats only if this currently doesnt satisfy the AC or you can think of an alternative.